### PR TITLE
SW-2239 Use consistent CRS for planting sites

### PR DESCRIPTION
**Pull request by sgrimm**
The default coordinate reference system (CRS) for GeoJSON objects is EPSG:4326,
which uses degrees of latitude and longitude. ArcGIS uses a spherical Mercator
coordinate system (EPSG:3857) by default, and we were returning geometry values
in that system which isn't usually what clients need.

Make two changes to the way we handle coordinates:

* Store coordinates in whatever coordinate reference system was used in the
  shapefiles we're importing.
* At read time, transform to EPSG:4326. (In the future, we might allow the client
  to ask for a different CRS.)

### CODE REVIEW

**Comment by sgrimm** [comment]
Current dependencies on/for this PR:
* main
  * **PR #794** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/794" target="_blank">Graphite</a> 
    * **PR #795** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/795" target="_blank">Graphite</a> 
      * **PR #796** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/796" target="_blank">Graphite</a> 
        * **PR #797** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/797" target="_blank">Graphite</a> 
          * **PR #798** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/798" target="_blank">Graphite</a> 
            * **PR #802** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/802" target="_blank">Graphite</a> 
              * **PR #803** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/803" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-server/803?utm_source=stack-comment).

**Comment by sgrimm** [comment]
Current dependencies on/for this PR:
* main
  * **PR #794** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/794" target="_blank">Graphite</a> 
    * **PR #795** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/795" target="_blank">Graphite</a> 
      * **PR #796** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/796" target="_blank">Graphite</a> 
        * **PR #797** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/797" target="_blank">Graphite</a> 
          * **PR #798** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/798" target="_blank">Graphite</a> 
            * **PR #802** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/802" target="_blank">Graphite</a> 
              * **PR #803** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/803" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-server/803?utm_source=stack-comment).

File: src/main/kotlin/com/terraformation/backend/search/field/GeometryField.kt [hunk 1]
```diff
@@ -24,7 +26,8 @@ class GeometryField(
   override val supportedFilterTypes: Set<SearchFilterType>
     get() = emptySet()
 
-  override val databaseField: Field<String?> = geometryField.asGeoJson()
+  override val databaseField: Field<String?> =
+      geometryField.transformSrid(SRID.LONG_LAT).asGeoJson()
```
Comment (sgrimm) [inline src/main/kotlin/com/terraformation/backend/search/field/GeometryField.kt:30]: `transformSrid()` is an extension function that already existed from our previous work on plant tracking: https://github.com/terraware/terraware-server/blob/main/src/main/kotlin/com/terraformation/backend/db/GeometryUtil.kt#L24

**Review by karthikbtf — APPROVED** [review]
Thanks for doing this.

**Pull request merged**